CHANGELOG.md

```diff
@@ -7,7 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-- n/a
+### Changed
+- Improved log message details in some cases of error
```

Comment:
Minor: How about adding "koji error" here?

Reply:
I was a little vague here because there are other pull requests also about improving logs in other cases of error - so I just wanted one note covering them all.  Unfortunately this setup having just one CHANGELOG file is quite annoying here, as separate PRs adding text at the same place will likely conflict, so there's a disincentive to write a separate point for every PR.